### PR TITLE
Partners

### PR DESCRIPTION
**Pull request by niallrobinson**

### CODE REVIEW

**Comment by niallrobinson** [comment]
currently doesn't link to the right place and I don't know why

File: _includes/partner.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!-- A Single team partner -->
+<div class="col-sm-6 col-md-3 col-lg-3">
+    <div class="team-member">
```
Comment (jacobtomlinson) [inline _includes/partner.html]: should probably have its own style class

Comment (niallrobinson) [inline _includes/partner.html]: hoped you wouldn't notice that

File: _sass/meto-informatics-lab.scss [hunk 2]
```diff
@@ -7,5 +7,6 @@
 @import "meto-informatics-lab/post";
 @import "meto-informatics-lab/team";
 @import "meto-informatics-lab/profile";
+@import "meto-informatics-lab/partner";
```
Comment (jacobtomlinson) [inline _sass/meto-informatics-lab.scss]: Have you actually added this file?

Comment (niallrobinson) [inline _sass/meto-informatics-lab.scss]: have now

**Pull request merged**